src/operator-docs/transformation/scan.ts

```diff
@@ -1,6 +1,62 @@
 import { OperatorDoc } from '../operator.model';
 
 export const scan: OperatorDoc = {
-  'name': 'scan',
-  'operatorType': 'transformation'
+  name: 'scan',
+  operatorType: 'transformation',
+  signature: 'public scan(accumulator: Function, seed: any): Observable',
+  parameters: [
+    {
+      name: 'accumulator',
+      type: '(acc: R, value: T, index: number) => R',
+      attribute: '',
+      description: 'The accumulator function called on each source value.'
+    },
+    {
+      name: 'seed',
+      type: 'T|R',
+      attribute: 'optional',
+      description: 'The initial accumulation value.'
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/scan.png',
```

Comment:
It looks like `scan` has an interactive marble diagram available (http://rxmarbles.com/#scan). Could you set `useInteractiveMarbles` property to true? This will use that on the page rather than the static image.